File: PathoFroh/src/main/java/com/patho/main/repository/TaskRepository.java
package com.patho.main.repository;

import java.util.Optional;

import com.patho.main.model.patient.Task;
import com.patho.main.repository.custom.TaskRepositoryCustom;

public interface TaskRepository extends BaseRepository<Task, Long>, TaskRepositoryCustom {
	public Optional<Task> findOptionalById(Long id);
}

File: PathoFroh/src/main/kotlin/com/patho/main/repository/TaskRepository.kt
package com.patho.main.repository

import com.patho.main.model.patient.Task
import com.patho.main.repository.custom.TaskRepositoryCustom
import java.util.*

interface TaskRepository : BaseRepository<Task, Long>, TaskRepositoryCustom {
    fun findOptionalById(id: Long?): Optional<Task>
}
